lib/services/screen-builder-service.ts

```diff
@@ -5,18 +5,24 @@ import * as path from "path";
 import * as util from "util";
 
 export class ScreenBuilderService implements IScreenBuilderService {
+	private static SCREEN_BUILDER_SPECIFIC_FILES = [".yo-rc.json", ".app.json", "app.js"];
+	private static UPGRADE_ERROR_MESSAGE = 'Your app has been build with an obsolete version of Screen Builder. Please, migrate it to latest version and retry.';
```

Comment:
I'm not sure about explaining about custom code migration in the error message, maybe in the command's help only?
What do you think @ikoevska